<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandArmTrainer</class>
 <widget class="QDialog" name="HandArmTrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arm/Hand Segment Trainer</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>1071</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_PlayStop">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_handPos"/>
    </item>
    <item>
     <widget class="QComboBox" name="cb_logTest"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_trainData">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>570</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_loadData">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>570</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Trained Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
